source/PlayState.hx

```diff
@@ -2,20 +2,40 @@ package;
 
 import Hero;
 import ScreensaverHero;
+import ControllableHero;
+import Wall;
+import flixel.FlxG;
+import flixel.FlxSprite;
 import flixel.FlxState;
 import flixel.text.FlxText;
 import flixel.util.FlxColor;
+import flixel.group.FlxGroup;
 
 class PlayState extends FlxState
 {
+	var controllableHero = new ControllableHero();
+	var screensaverHeroes = new FlxTypedGroup<ScreensaverHero>();
+	var walls = new FlxTypedGroup<Wall>();
+
 	override public function create()
 	{
 		super.create();
 
-		add(new ScreensaverHero());
-		add(new ScreensaverHero());
-		add(new ScreensaverHero());
-		add(new ScreensaverHero());
+		screensaverHeroes.add(new ScreensaverHero());
+		screensaverHeroes.add(new ScreensaverHero());
+		screensaverHeroes.add(new ScreensaverHero());
+		screensaverHeroes.add(new ScreensaverHero());	
+
+		add(screensaverHeroes);
```

Comment:
Loops are your friend here, something like:
```
for (i in 0...4) {
    add(new ScreensaverHero())
}
```
The walls below are trickier, but there are a variety of ways you could handle generating them if you wanted to be a bit more programmatic about it (keep a list of starting coordinates for the walls you want to generate, for instance).

This gets the job done, which is a fine result when you're under time pressure, but it's not hard to see the pain that repetition brings. That, and it doesn't hurt to get more comfortable with the fundamentals of the Haxe language.

No action required here, but optionally swap to a loop if you want the practice.